mapper/mapper.go

```diff
@@ -253,7 +253,7 @@ func CreateCalendar(_ context.Context, basePage coreModel.Page, _ config.Config)
 		Page: basePage,
 	}
 	calendar.BetaBannerEnabled = true
-	calendar.Metadata.Title = "Release Calendar"
```

Comment:
Uhm... shouldn't we be doing the localisation in the template, not the model?

Reply:
That's a good point.

`Metadata.Title` is rendered by `assets/templates/main.tmpl` from `dp-renderer`, which consumes it directly instead of through localisation:

```html
<!DOCTYPE html>
<html lang="{{if .Language }}{{ .Language }}{{ else }}en{{ end }}" xml:lang="{{ if .Language }}{{ .Language }}{{ else }}en{{ end }}">
  <head>
    <title>
      {{ if .Error.Title }}
        {{ .Error.Title }}
      {{ else if .Metadata.Title }}
        {{ .Metadata.Title }}
      {{- end }}
      - {{ localise "OfficeForNationalStatistics" .Language 1 }}
    </title>

```

With `dp-renderer` being common to all frontends, so far I can think of two choices:
- supply the localised title to dp-renderer (as in this PR), or
- add a list of page title localisations to dp-renderer's locales for every page in every frontend that uses dp-renderer

Reply:
Sorry I didn't realise this was being used by a common template. I think it's fine this way then 👍